reactor-core/src/test/kotlin/reactor/core/publisher/MonoExtensionsTests.kt

```diff
@@ -190,7 +191,7 @@ class MonoExtensionsTests {
     fun onErrorResume() {
         StepVerifier.create(IOException()
                 .toMono<String>()
-                .onErrorResume(IOException::class, { "foo".toMono() }))
+                .onErrorResume(IOException::class) { "foo".toMono() })
```

Comment:
Same issue as in `MonoFunctionsTests` - reads as `(IOException::class) { "foo".toMono() }` 😊